packages/fxa-settings/src/Settings/components/App/en-US.ftl

```diff
@@ -0,0 +1,45 @@
+## Firefox and Mozilla Brand
```

Comment:
It seems like all files besides this one are just being moved. It appears this file in particular is being added, but the old file is also being left in place at`fxa-settings/src/components/App/en-us.ftl`. Is this intentional?

Also, since I'm not 100% certain, can you confirm that the grunt script will pickup on the new paths? (I'd like to avoid a situation where this gets out of sync with Pontoon.)